meetup_api.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import logging
+
+MEETUP_BASE_URL = "https://www.meetup.com"
```

Comment:
The official documentation suggests the use of `[api.meetup.com](https://api.meetup.com/gql)`. Ref: https://www.meetup.com/api/guide/#graphQl-guide

Reply:
Does this work without an API key?

Reply:
Nope. It’s better to use this endpoint.

Reply:
This as in the current internal API? I don't think we can get a key for the official API since they restrict it to PRO accounts now.

Reply:
@ditsuke We can created an OAuth application and then get the API key once it's approved - https://www.meetup.com/api/oauth/list/

Reply:
As @zora89 mentioned he has already given it a go and been rejected 😞